frontend/src/metabase/actions/components/ActionForm/ActionForm.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { useMemo } from "react";
+import { t } from "ttag";
+import _ from "underscore";
+import { DragDropContext, Droppable, Draggable } from "react-beautiful-dnd";
+
+import type {
+  DraggableProvided,
+  OnDragEndResponder,
+  DroppableProvided,
+} from "react-beautiful-dnd";
+
+import Button from "metabase/core/components/Button";
+import Form from "metabase/core/components/Form";
+import FormProvider from "metabase/core/components/FormProvider";
+import FormSubmitButton from "metabase/core/components/FormSubmitButton";
+import FormErrorMessage from "metabase/core/components/FormErrorMessage";
+
+import type {
+  ActionFormSettings,
+  FieldSettings,
+  WritebackParameter,
+} from "metabase-types/api";
+
+import type { Parameter } from "metabase-types/types/Parameter";
+
+import { reorderFields } from "metabase/actions/components/ActionCreator/FormCreator";
+import Icon from "metabase/components/Icon";
```

Comment:
Think this import should go right after `metabase/core/components` imports

Reply:
thanks. we gotta figure out how to make the linter do this 🤔